cpp/src/lib/Invoice.cpp

```diff
@@ -41,6 +41,7 @@ double purchase::Invoice::computeTotalAmount() const
 	for (const auto purchasedBook : purchasedBooks_)
 	{
 		double totalPrice = purchasedBook->getTotalPrice();
+		//double totalPrice = purchasedBook->getTotalPrice() * finance::getApplicableRate(country_, *purchasedBook->getBook());
```

Comment:
Shouldn't we add a TOD comment here like you did in ReportGenerator ?
//TODO Uncomment to fix the bug in Invoice

Reply:
You are right, it would be better to do so! 
I will fix this